content/docs/user-guide/networking/multiplayer/autocomponents.md

```diff
@@ -9,47 +9,132 @@ weight: 400
+| IsPublic | If `true`, this property's access modifier is set to `public`, and is accessible from outside of the class.  | `bool` |
+| IsRewindable |  | `bool` |
+| IsPredictable |  | `bool` |
+| ExposeToEditor | If `true`, this property is accessible in the O3DE Editor. | `bool` |
+| ExposeToScript | If `true`, this property is accessibly via script.  | `bool` |
+| GenerateEventBindings |  | `bool` |
+
+### Remote procedure calls (RPCs)
+
+RPCs have two important properties: `InvokeFrom` and `HandleOn`. These properties describe which role the RPC is invoked from and which role it's handled on.
+
+The following are four types of RPCs with a valid "invoked from and handled on" relationship and their possible use cases:
+
+- **Authority-to-Autonomous**: In an example use case, it sends corrections about the game state to the user.
+
+- **Authority-to-Client**: Authority sends calls to all clients. For example, it sends information about particle effects.
+
+- **Server-to-Authority**: This is required to communicate information between entities. For example, suppose in a multi-server setup, EntityA is owned by ServerA and EntityB is owned by ServerB. If EntityA communicates directly to EntityB, EntityA will be talking to a proxy, not the real EntityB. Server-to-Authority ensures that messages always find the entity with authority. When a player wants to deal damage, Autonomous informs the Server, and the Server sends the DealDamage function to Authority. 
+
+- **Autonomous-to-Authority**: Sends user settings information that affects user input and is used during input-process time rather than input-creation time, such as mouse sensitivity and input controls. 
+
+The `RemoteProcedure` tag has the following properties:
+
+| Property | Description | Values | 
+| - | - | - |
+| Name | Name of the RPC. | `string` |
+| Param | A parameter and value pair that you can send along with the RPCs. You can have none or an infinite amount of parameters. You must specify the parameter's type and name. |  |
+| Description | Describes the functionality of the RPC. | `string` |
+| InvokeFrom | Tells the network which role invokes this RPC. | `Server`, `Autonomous`, `Client` | 
+| HandleOn | Tells the network which roles handles this RPC. If Authority, the RPC is handled by the server that has authority over that entity. If Autonomous, the RPC is handled only by the relevant player's local client. If Client, the RPC is handled on all clients. | `Authority`, `Autonomous`, `Client` |
+| IsPublic | If `true`, this property's access modifier is set to `public`, and is accessible from outside of the class.  | `bool` |
+| IsReliable | If `true`, RPC is reliable and uses a queue to guarantee the delivery of a message. Otherwise, RPC is unreliable and are sent over a "fire and forget" method. By default, RPCs are reliable. | `bool` |
+| GenerateEventBindings |  | `bool` |
```

Comment:
If true a callback (see AZ::Event doc) will be triggered whenever this RPC is handled. Game logic outside of this component's class can subscribe and be notified whenever the RPC occurs.